### DISCUSSION TITLE
3.14版本115网盘受限

### DISCUSSION THREAD

**Discussion by asygr** [AlistGo/alist#3946] [General]
3.14版本115网盘受限

**Comment by anwen-anyi** [comment]
修复了等着下个稳定版发布就行，要是急急国王着急的要么自己降版本 要么用测试版
https://github.com/alist-org/alist/pull/3932

**Comment by WUFXXX** [comment]
win桌面端自动更新了，115只能浏览文件列表无法访问文件，无法播放视频下载文件，手机端3.13正常使用，升级3.14后同上无法访问，希望尽快发布稳定版吧
